### ISSUE TITLE
Map.getLicenseInfo() = undefined

### ISSUE THREAD

**Issue by hisshash** [mapsplugin/cordova-plugin-googlemaps#1073]
Hey,

I've added a button to call the getLicenseInfo function which is appending undefined but with no errors etc.
This is happening on my Android
```
map.getLicenseInfo(function(txt, txt1) {
  var div = document.getElementById('licenseView');
  div.innerText = txt || txt1;
});
```

Not experiencing any others problems. Great plugin.

Cordova -v 6.3.0
Cordova Android -v 5.2.1
Cordova GoogleMaps -v Latest

Cheers

**Comment by wf9a5m75** [comment]
https://github.com/mapsplugin/cordova-plugin-googlemaps/issues/975

**Comment by hisshash** [comment]
Ah missed that, Thanks man

**Comment by apptum** [comment]
I'm testing on Android 7.1.1, Play Services 10.2.98, maps plugin 1.3.9, and still get `undefined`.
Any tipps to solve that issue? Is it save to link to a "static" copy like in #975?

According to Google, it should be resolved:
> This issue is resolved the 9.4 Google Play Services client library, so we recommend you update to that version.
https://code.google.com/p/gmaps-api-issues/issues/detail?id=9813
